src/SortedArray.hpp

```diff
@@ -227,40 +208,23 @@ class SortedArray : protected SortedArrayView< T, INDEX_TYPE >
    * @return True iff the value was actually removed.
    */
   inline
-  bool erase( T const & value ) LVARRAY_RESTRICT_THIS
+  bool remove( T const & value ) LVARRAY_RESTRICT_THIS
```

Comment:
Why the name change? `erase` is the same as `std::set`...not that this is a good reason to keep `erase`.

Reply:
`insert` and `remove` seems like a better match to me. `erase` sounds good for when you want to delete a position whereas you `remove` a value. But it's all pretty trivial.